docker/README.md

```diff
@@ -2,19 +2,17 @@
 
 ## Prerequisites
 - install docker
-- install docker compose (probably bundled with docker these days)
+- install docker compose (probably bundled with your version of docker)
 - run on linux :) (other platforms may work but YMMV)
 
-> [!TIP]
-> If you're just running a local network for testing, you probably want to use the local non-TLS image.
-> For that, use ghcr.io/drand/go-drand-local instead
-
 ## Creation of a keypair
 
 Pull the latest drand image:
 ```shell
-docker pull ghcr.io/drand/go-drand:v2.0.7-testnet
+docker pull ghcr.io/drand/go-drand:v2.0.1
 ```
+> [!NOTE]
+> If you want to run drand locally without TLS, you should use the `ghcr.io/drand/go-drand-local:v2.0.1` image instead!
```

Comment:
I think this would warrant a github action step to update this text in our publish image GHA, WDYT?

Reply:
yep, good idea!